scm/src/scm.F90

```diff
@@ -82,8 +82,6 @@ subroutine scm_main_sub()
   select case(scm_state%time_scheme)
```

Comment:
if `time_scheme` only has one possible value (1), why don't we get rid of it altogether and remove this select/case statement?